contracts/Colony.sol

```diff
@@ -277,6 +281,15 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     emit ColonyUpgraded(currentVersion, _newVersion);
   }
 
+  function finishUpgrade2To3() public always {
+    ColonyAuthority(address(authority)).setRoleCapability(
+      uint8(ColonyDataTypes.ColonyRole.Root),
+      address(this),
+      bytes4(keccak256("updateColonyOrbitDB(string)")),
+      true
+    );
+  }
```

Comment:
For atomicity I feel this function should be internal and called from within "upgrade"

Reply:
I I don't think it can be internal and called from within `upgrade` because it doesn't exist on v2 Colony.sol, which we'll still be `delegatecall`'d in to even after calling `.setResolver`. We can however keep it external, perhaps even make it colony only, and do an external call after setting the new resolver.

Reply:
Called conditionally when `currentColonyVersion = 2`?